tests/tests.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+current_directory=$(pwd)
+test_framework_directory="$current_directory/jenkinsfile-runner-test-framework"
+sh_unit_directory="$test_framework_directory/shunit2"
+
+version="256.0-test"
+CWP_version="1.3"
+jenkinsfile_runner_tag="jenkins-experimental/jenkinsfile-runner-test-image"
+
+downloaded_cwp_jar="to_update"
+
+. $test_framework_directory/utilities/utils.inc
```

Comment:
I think the framework should be sourced by a single command, maybe you could add a wrapper init script in https://github.com/jenkinsci/jenkinsfile-runner-test-framework/pull/1